lighthouse-core/gather/driver.js

```diff
@@ -705,6 +709,36 @@ class Driver {
     };
   }
 
+  /**
+   * Returns a promise that resolve when a frame has a LCP.
```

Comment:
```suggestion
   * Returns a promise that resolves when a frame has a LCP.
```

Reply:
heh, also in the original method where this was copied from :)